src/js/ProgressBar.js

```diff
@@ -53,25 +53,25 @@ export default class ProgressBar extends React.Component {
             default:
                 break
         }
+        var timeSince = new Date((now - this.props.updateTime) * this.props.countRate)
 
         if (this.props.countDirection === "COUNTDOWN") {
             var startPosition = new Date(startDate.getTime() - offset)
-            var timeSince = new Date((now - this.props.updateTime) * this.props.countRate)
-            var position = new Date(startPosition  - timeSince)
+            // Start position is used if paused
+            var position = this.props.paused ? startPosition : new Date(startPosition - timeSince)
```

Comment:
```suggestion
            var position = this.props.paused ? startPosition : new Date(startPosition.getTime() - timeSince.getTime())
```
For the sake of consistency, it doesn't make a difference here but we're using it in all other places for this PR